b9lab-content/5-my-own-chain/03-starport.md

```diff
@@ -1 +1,210 @@
-# Starport
\ No newline at end of file
+# Starport
+
+## Install
+
+The quickest and simplest way to create blockchain applications with the Cosmos SDK is to use Starport. 
+
+To install Starport at the command line:
+
+```bash
+$ curl https://get.starport.network/starport! | bash
+
+```bash
+$ sudo mv starport /usr/local/bin/
```

Comment:
```suggestion
```

If you're using `!` as part of the URL above, you don't need to move the binary manually. If `!` doesn't work by default (permissions issue), refer to the install guide: https://docs.starport.com/guide/install.html